README.md

```
@@ -1,202 +1,64 @@
+
+VAULT_ADDR (e.g., http://localhost:8200)
+VAULT_ROLE_ID
+VAULT_SECRET_ID
+(Optional) ANTHROPIC_API_KEY as a fallback
+Usage Example
+
+Python
+from secure_analyzer import summarize, display_summary
+
+# Summarize a website and print result
+summary = summarize("https://devopsmayur.github.io/mgblogs")
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)